client/components/phone-input/index.jsx

```diff
@@ -39,6 +42,12 @@ const PhoneInput = React.createClass( {
 		};
 	},
 
+	componentWillReceiveProps( nextProps ) {
+		if ( this.props.value !== nextProps.value && nextProps.value ) {
```

Comment:
This looks like it could be a typo?
Is this intentional or should this be `nextProps && nextProps.value` ?